content/docs/reference/pkg/nodejs/pulumi/kafka/_index.md

```diff
@@ -0,0 +1,774 @@
+---
+title: "Package @pulumi/kafka"
+linktitle: "@pulumi/kafka"
+meta_desc: "Explore members of the @pulumi/kafka package."
+---
+
+<!-- WARNING: this page was generated by a tool. Do not edit it by hand. -->
+<!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/tscdocgen. -->
+
+
+> This provider is a derived work of the [Terraform Provider](https://github.com/terraform-providers/terraform-provider-kafka)
+> distributed under [MPL 2.0](https://www.mozilla.org/en-US/MPL/2.0/). If you encounter a bug or missing feature,
+> first check the [`pulumi/pulumi-kafka` repo](https://github.com/pulumi/pulumi-kafka/issues); however, if that doesn't turn up anything,
+> please consult the source [`terraform-providers/terraform-provider-kafka` repo](https://github.com/terraform-providers/terraform-provider-kafka/issues).
```

Comment:
So... When we generate the provider, we emit a README.md in our provider repo's node sdk folder that contains this text (which we then include here in the docs...)

Since this provider isn't coming fro the https://github.com/terraform-providers, I think we'll need to manually edit the text of that README file to point to the real repo. Otherwise, this is a link to a non-existent repo.

Reply:
I can't see https://github.com/pulumi/pulumi-kafka yet (currently 404s for me), so can't point you to the exact file that needs to be changed.

Reply:
BTW, @stack72, do you know if we have any other TF-based providers that don't come from https://github.com/terraform-provider? If so, those should be fixed-up as well.

Reply:
https://github.com/pulumi/pulumi-kafka/blob/master/sdk/nodejs/README.md